data/botPolicies.json

```diff
@@ -588,6 +588,17 @@
         "91.242.162.0/24"
       ]
     },
+    {
+      "name": "internet-archive",
+      "action": "ALLOW",
+      "remote_addresses": [
+        "207.241.224.0/20",
+        "207.241.224.0/24",
```

Comment:
```suggestion
```
that is overlapping with 207.241.224.0/20

Reply:
You're very much right, thank you very much, I will fix that~

Reply:
207.241.237.0/24 was also overlapping (207.241.224.0/20 goes up to 207.241.239.255), so I removed it~